modules/controlled_access_terms_defaults/config/install/core.entity_view_display.taxonomy_term.resource_types.default.yml

```diff
@@ -0,0 +1,35 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.taxonomy_term.resource_types.field_external_authority_link
+    - taxonomy.vocabulary.resource_types
+  module:
```

Comment:
Same as a comment above, this should include controlled_access_terms in the module dependencies.

Reply:
Added - see line below this